docs/doxygen/overviews/xrc_format.h

```diff
@@ -907,6 +907,21 @@ pseudo-class. @c ownerdrawnitem objects have the following properties:
     Label to display on the button (may be omitted if only the bitmap or stock ID is used).}
 @row3col{bitmap, @ref overview_xrcformat_type_bitmap,
     Bitmap to display in the button (optional).}
+@row3col{pressed, @ref overview_xrcformat_type_bitmap,
+     Bitmap to show when the button is pressed (default: none, same as @c bitmap).
+     This property exists since wxWidgets 3.1.6, but the equivalent (and still
```

Comment:
Is it correct to speak about the old property here? AFAICS it was never supported for `wxButton`, so this is misleading. IMHO we should just say that this property is new in 3.1.7.